source/MRViewer/MRViewer.cpp

```diff
@@ -2026,6 +2025,30 @@ void Viewer::postSetIconified( bool iconified )
 
 void Viewer::postFocus( bool focused )
 {
+#ifdef _WIN32
+    if ( focused && ImGui::isMultiViewportEnabled() )
+    {
+        std::vector<GLFWwindow*> processedWindow;
+        for ( ImGuiWindow* win : ImGui::GetCurrentContext()->Windows )
+        {
+            if ( !win->Viewport )
+                continue;
+            GLFWwindow* glfwWindow = ( GLFWwindow* )win->Viewport->PlatformHandle;
+            if ( !glfwWindow || getViewerInstance().window == glfwWindow )
+                continue;
+
+            auto findIt = std::find( processedWindow.begin(), processedWindow.end(), glfwWindow );
+            if ( findIt != processedWindow.end() )
+                continue;
+
+            processedWindow.push_back( glfwWindow );
+            {
+                HWND hwnd = glfwGetWin32Window( glfwWindow );
+                SetWindowPos( hwnd, HWND_TOP, 0, 0, 0, 0, SWP_NOMOVE | SWP_NOSIZE | SWP_NOACTIVATE );
+            }
+        }
+    }
+#endif
```

Comment:
its better to move this logic to ImGuiMenu

Reply:
moved